app/main/presenters/search_summary.py

```diff
@@ -266,4 +266,8 @@ def _mark_up_filter(filter):
                     filter_string = self.rules.add_filter_preposition(
                         filter_id=filter, filter_string=filter_string)
                     processed_filters.append(filter_string)
+                else:
+                    raise Exception(
+                        "Filter with id '{}' not found in filterRules.".format(filter)
```

Comment:
Yeah, totally sensible, I thought someone might ask me to write a test for the new behaviour though :-D